libraries/aws_cloudfront_distribution.rb

```diff
@@ -13,12 +13,12 @@ class AwsCloudFrontDistribution < AwsResourceBase
   "
 
   attr_reader :distribution_id, :viewer_certificate_minimum_ssl_protocol, :viewer_protocol_policies,
-              :custom_origin_ssl_protocols, :s3_origin_configs, :custom_origin_protocol_policies
+              :custom_origin_ssl_protocols, :s3_origin_configs, :custom_origin_protocol_policies, :origin_domain_name, :s3_origin_path
```

Comment:
now after refactoring, `origin_domain_name` is obsolete right

Reply:
This is the input param, which required to pass .
```
describe aws_cloudfront_distribution(distribution_id: 'ABC', origin_domain_name: 'test.s3.us-east-1.amazonaws.com') do
        its ('s3_origin_path') { should include '/path' }
      end
```

Reply:
No, but you have an attr_reader

Reply:
fixed